revise doc (#4406)
istio/istio.io: content/docs/tasks/traffic-management/ingress/secure-ingress-sds/index.md

`httpbin.example.com` and `helloworld-v1.example.com`, for example. The ingress gateway
retrieves unique credentials corresponding to a specific `credentialName`.

> 1.  Restore credentials for httpbin by deleting its secret and creating again.

Reviewer: ```suggestion
1.  To restore the credentials for `httpbin`, delete its secret and create it again.
```
